setup.py

```diff
@@ -36,6 +36,10 @@ def get_install_requires():
             continue
         # add line to requirements
         requirements.append(line.replace('\n', ''))
+    if sys.version_info.major == 3:
```

Comment:
use "major than or equal to" instead of "equal to" please

Reply:
Ok Changed it